### ISSUE TITLE
Remove shell color codes from log output option?

### ISSUE THREAD

**Issue by pamubay** [thecoshman/http#114]
I want to write the log to a file.
Using simple shell redirection `http > ./log.txt`, unfortunately produces ugly log lines, as it's mixed with shell color codes.

```
[00mHosting "." on port 8000 without TLS and no authentication...
Ctrl-C to stop.

[36m[2020-09-27 09:53:02][00m [32m127.0.0.1:35120[00m was served file [35m/home/pamubay/dev/projects/tests/index.html[00m as [34mtext/html[00m[30m[31m[32m[33m[34m[35m[36m[37m[00m
[36m[2020-09-27 09:53:02][00m                           found index file for directory [35m/home/pamubay/dev/projects/tests[00m[30m[31m[32m[33m[34m[35m[36m[37m[00m
```
a new option `--no-color` or anything to produce clean output would be nice.

I'm on Linux

Thanks.

**Comment by nabijaczleweli** [comment]
Can you try with current `develop` (at least 266ea83608d31272e3696be81dd74fa19e948aa2) and the `--no-colour` option?

**Comment by pamubay** [comment]
The output still has color codes left, on the first line.

```
[00mHosting "." on port 8000 without TLS and no authentication...
Ctrl-C to stop.

[2020-09-27 19:56:59] 127.0.0.1:52474 was served directory listing for /home/pamubay/dev/repos/http/target/debug/build
[2020-09-27 19:57:00]                 encoded as gzip for 426.0% ratio
[2020-09-27 19:57:02] 127.0.0.1:52474 was served directory listing for /home/pamubay/dev/repos/http/target/debug
[2020-09-27 19:57:02]                 encoded as gzip for 347.7% ratio
[2020-09-27 19:57:04] 127.0.0.1:52474 was served directory listing for /home/pamubay/dev/repos/http/target/debug/deps
```

Also, if it's possible, can we have short flag variants?
Thank you.

**Comment by nabijaczleweli** [comment]
Good catch, 00f19c554792e05c7991b095c844963f1c94cee3 should fix both the initial reset and adds `-c` as the short option.

**Comment by pamubay** [comment]
Tested, Works perfectly. :+1: 

Thank You.

**Comment by nabijaczleweli** [comment]
Great; released in [`v1.10.0`](https://github.com/thecoshman/http/releases/tag/v1.10.0)!
